apps/src/javalab/Neighborhood.js

```diff
@@ -0,0 +1,35 @@
+const maze = require('@code-dot-org/maze');
```

Comment:
should this filename be capitalized? if it's a collection of helper functions, we usually use lowercase; for react components and classes, we capitalize (although some files like [applab.js](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/applab/applab.js) don't follow that convention, but should be updated)

Reply:
Hmm I capitalized it because it should be thought of as a widget or a mini-app. It probably makes sense to go ahead and make it a class. I'll do that.

Reply:
✔️